assets/src/application/services/utilities/memo/getPropsAreEqual.ts

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { path } from 'ramda';
+
+type PropsAreEqual<P extends {}> = (
+	prevProps: Readonly<React.PropsWithChildren<P>>,
+	nextProps: Readonly<React.PropsWithChildren<P>>
+) => boolean;
+
+/**
+ * Generates the comparison function that can be used as second argument to React.memo()
+ */
+const getPropsAreEqual = <P extends {}>(pathToValue: Parameters<typeof path>[0]): PropsAreEqual<P> => {
```

Comment:
is there a comparison measurement with and without this function?

Reply:
`React.memo` does not necessarily prevent re-renders, it's just a performance optimization. I haven't measured the difference.